client/extensions/woocommerce/woocommerce-services/views/label-settings/label-payment-method.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormCheckbox from 'components/forms/form-checkbox';
+import CompactCard from 'components/card/compact';
+import PaymentLogo from 'components/payment-logo';
+
+export const getPaymentMethodTitle = ( translate, paymentType, digits ) => {
+	const supportedTypes = {
+		amex: translate( 'American Express' ),
+		discover: translate( 'Discover' ),
+		mastercard: translate( 'MasterCard' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Visa, MasterCard' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).